agenta-cli/agenta/cli/helper.py

```diff
@@ -153,7 +158,7 @@ def update_config_from_backend(config_file: Path, host: str):
     assert config_file.exists(), "Config file does not exist!"
     config = toml.load(config_file)
     app_id = config["app_id"]
-    api_key = config.get("api_key", None)
+    api_key = config.get("api_key", "")
```

Comment:
why the change

Reply:
While using Fern, it the api key is passed as having a value of None, it raises an error.